apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/AutomatiskBehandlingRoutes.kt

```diff
@@ -42,13 +43,27 @@ fun Route.automatiskBehandlingRoutes(
                 behandlingKlient.tildelSaksbehandler(oppgaveTilAttestering, brukerTokenInfo)
 
                 logger.info("Attesterer vedtak for behandling $behandlingId")
-                service.attesterVedtak(
-                    behandlingId,
-                    "Automatisk attestert av ${Fagsaksystem.EY.systemnavn}",
-                    brukerTokenInfo,
-                )
+                val attestert =
```

Comment:
Hva er end game for routsa, dette er jeg ikke fan å legge her.

Reply:
Heile try-blokka under skal ut til vedtaksvurdering-kafka.